File: s3-trigger-slack-app/SendToSlack/slack.py
import json

from botocore.vendored import requests

def sent_slack_message(data):
    JSON_HEADER = {'Content-Type': 'application/json'}
    url = AddUrlHook
    
    response = requests.post(url, data=json.dumps(data), headers=JSON_HEADER)
    
    return response

File: s3-trigger-slack-app-scala/src/main/scala/aws/lambda/slack.scala
package aws.lambda

import scalaj.http.{Http, HttpOptions}


object slack {

  def sent_slack_message(data: String):Int = {
    val url = "SlackHook"

    val result = Http(url).postData(data)
      .header("Content-Type", "application/json")
     // .header("Charset", "UTF-8")
      .option(HttpOptions.readTimeout(10000)).asString
    result.code

  }

}
